### DISCUSSION TITLE
Somehow yabai keeps appearing in macOS settings after being uninstalled

### DISCUSSION THREAD

**Discussion by Rhamao** [asmvik/yabai#1047] [Q&A] [answered]
Recently I've installed Yabai on macOS Big Sur to try it out. But I finnaly end up using another window manager. So I uninstalled yabai following the steps indicated in the wiki. But after that Yabai keep appearing in [System Preferences>Security & Privacy>Accessibility] :
![image](https://user-images.githubusercontent.com/47444260/141705989-2b79b923-015a-4698-9b48-2b49324bbb60.png)
I decided to look for all files and folders that I could find containing the characters "yabai" (in their name or in their content). I only found homebrew configuration files (except for some airports.dat files containing coincidentally this characters : "Ahilyabai Holkar Airport" ). 

Deleting them had no effect as I expected. So I reinstalled the deleted homebrew configuration files. Followed again the wiki install/uninstall steps. But still nothing changed, yabai was still appearing in [System Preferences>Security & Privacy>Accessibility].

I'll also try restarting my computer if someone is wondering.

So if anyone ran through the same issues or has an idea of how I could change that, I'll be happy to hear from you :).

**Comment by es183923** [comment, marked as answer]
Yabai is not checked, meaning that it does not have access. All apps that have requested the Accessibility permission would show up there, but only the ones that are checked actually have the permission.

The grey icon that yabai has means that yabai that requested the permission, and was removed.

To remove it, just unlock it, click on yabai, and press the `-` button. It'll remove yabai from showing up there.

**Comment by Rhamao** [comment]
That was the solution. I think constantly changing OS made me blind to what was an obvious solution.  Anyway I am glad you could answer this :). I look to close and marked as resolved this discussion.

**Reply by Rhamao** [reply]
I have marked you're comment as an "Answer". I don't see an option to close the discussion so far.

**Comment by Gadou0099** [comment]
Hello 🤗
